tests/foreman/cli/test_host_collection.py

```diff
@@ -64,43 +65,99 @@ def _make_fake_host_helper(self):
             u'organization-id': self.organization['id'],
         })
 
+    @upgrade
     @tier1
```

Comment:
I think more related tier is now tier2/3 ? could you please check.

Reply:
Good point. Updated